app.go

```diff
@@ -193,47 +209,48 @@ type Static struct {
 	// When set to true, the server tries minimizing CPU usage by caching compressed files.
 	// This works differently than the github.com/gofiber/compression middleware.
 	// Optional. Default value false
-	Compress bool
+	Compress bool `json:"compress"`
 
 	// When set to true, enables byte range requests.
 	// Optional. Default value false
-	ByteRange bool
+	ByteRange bool `json:"byte_range"`
 
 	// When set to true, enables directory browsing.
 	// Optional. Default value false.
-	Browse bool
+	Browse bool `json:"browse"`
 
 	// The name of the index file for serving a directory.
 	// Optional. Default value "index.html".
-	Index string
+	Index string `json:"index"`
 }
 
-// default settings
+// Default Config values
 const (
-	defaultBodyLimit            = 4 * 1024 * 1024
-	defaultConcurrency          = 256 * 1024
-	defaultReadBufferSize       = 4096
-	defaultWriteBufferSize      = 4096
-	defaultCompressedFileSuffix = ".fiber.gz"
+	DefaultBodyLimit            = 4 * 1024 * 1024
+	DefaultConcurrency          = 256 * 1024
+	DefaultReadBufferSize       = 4096
+	DefaultWriteBufferSize      = 4096
+	DefaultCompressedFileSuffix = ".fiber.gz"
```

Comment:
These are the only config fields that have a different default then all the others, I don't see a reason for not having them. Could be useful to someone for debugging or to peek the value that is set by default in case they want to override it.

Feel free to change my mind!

Reply:
My reservations come from the added maintenance of a bigger package API. E.g.

1.  If in the future we add a new DefaultFoo value to a new configuration option, should it also be exposed? This approach would imply yes.
1. Removing/renaming any of the values requires a new major release, which (as I'm sure you're starting to realise) is a lot of work.

However, I buy the argument of including them for the purpose of documentation. 👍 I'm not willing to die on this hill!